src/ribbon/buttonbar.cpp

```diff
@@ -978,8 +1004,10 @@ void wxRibbonButtonBar::CommonInit(long WXUNUSED(style))
     // Create() later.
     m_ribbonBar = GetAncestorRibbonBar();
 
-    m_bitmap_size_large = wxSize(32, 32);
-    m_bitmap_size_small = wxSize(16, 16);
+    // Note: size definition before 3.1.7 was always overriden by first added button
+    // in wxRibbonButtonBar::InsertButton()
+    m_bitmap_size_large = wxSize(0, 0);
+    m_bitmap_size_small = wxSize(0, 0);
```

Comment:
Do we actually need to do this at all? They're (0,0) by default. If do initialize them, it would arguably make more sense to initialize them to `wxDefaultSize`.

I also don't think we need such historical comments as the one above, we have the VCS history for this.

Reply:
I can delete it. It was just added for clarifying, why I removed the old "default" values. Initializing with 0,0 (by default) is in my opinion the best way, because it doesn't change actual default behavior and additional with `SetBitmapSize()` the developer can decide if we wants to use the default size or another size.